base/src/test/kotlin/proguard/analysis/PartialEvaluatorErrorsTest.kt

```diff
@@ -67,6 +68,34 @@ class PartialEvaluatorErrorsTest : FreeSpec({
             }
         }
 
+        "Changing stack size" {
+            val programClass = buildClass()
+                .addMethod(AccessConstants.PUBLIC, "test", "()I", 50) {
+                    val startLabel = it.createLabel()
+                    val elseLabel = it.createLabel()
+                    it.iconst(50)
+                        .label(startLabel)
+                        .dup()
+                        .iconst_5()
+                        .ifle(elseLabel) // To be working code, this should have been if_icmple
```

Comment:
Can you place the comment on top of the instruction and explain it in a little more detail?